src/clientsettingsdlg.cpp

```diff
@@ -253,6 +253,24 @@ CClientSettingsDlg::CClientSettingsDlg ( CClient*         pNCliP,
     edtNewClientLevel->setWhatsThis ( strNewClientLevel );
     edtNewClientLevel->setAccessibleName ( tr ( "New client level edit box" ) );
 
+    // input gain
+    QString strInputGain = "<b>" + tr ( "Input Gain" ) + ":</b>" +
+        tr ( "This setting allows to increase your input signal level. "
+        "If your sound is too quiet, first try to increase the level by "
+        "getting closer to the microphone, adjusting your sound equipment "
+        "or increasing levels in your operating system's input settings. "
+        "Only if this fails, set a value above 100% here. "
+        "If your sound is too loud, sounds distorted and is clipping, this "
+        "option will not help. Do not use it. The distortion will still be "
+        "there. Instead, decrease your input level by getting farther away "
+        "from your microphone, adjusting your sound equipment "
+        "or by decreasing your operating system's input settings.<br />"
+        "Valid range: 100% (default) - 1000% (ten-time increase)"
```

Comment:
Can you add
`"<br/> + "(In decibels: 0dB to +20dB)"`
(for those who think in decibels).

Reply:
Thanks, done.